tests/integration/test_github.py

```diff
@@ -129,6 +129,13 @@ def test_username(self):
         # changed to check just lenght, because it is based who regenerated data files
         assert len(self.service.user.get_username()) > 3
 
+    def test_email(self):
+        test_str = self.service.user.get_email()
+        assert test_str != None
```

Comment:
```suggestion
        assert test_str
```
This should be enough, and you will also fix the problem found by `pre-commit`.